ports/curl/portfile.cmake

```diff
@@ -29,7 +31,6 @@ set(BUILD_OPTIONS
     # CMAKE options
     -DCMAKE_USE_GSSAPI=OFF
     -DCMAKE_USE_LIBSSH2=OFF
-    -DCMAKE_USE_OPENLDAP=OFF

```

Comment:
Is this related to static builds or is it a drive-by fix?

Reply:
It was a warning saying it was unused when doing a build. Can move to a separate commit.